oeps/oep-0010.rst

```diff
@@ -38,8 +38,8 @@ process.
 Motivation
 ==========
 
-Open edX releases happen infrequently, every four to six months. EdX engineers
-are typically focused on the weekly edx.org releases.  This document
+Open edX releases happen roughly every six months. EdX engineers
+are typically focused on the daily edx.org releases.  This document
```

Comment:
It's not really daily, depends on which system we're talking about - we do multiple releases per day in aggregate. I'd just remove the word entirely to avoid unnecessary complication ("focused on the edx.org releases").